### ISSUE TITLE
manager_start script's output should be redirected to application's logs

### ISSUE THREAD

**Issue by marcoboi** [spark-jobserver/spark-jobserver#916]
No logging is produced when the manager_script is run.
Moreover, the logging of the spark submit command is redirected to {{null}}, making any error in the script difficult to catch.

https://github.com/spark-jobserver/spark-jobserver/blob/eaebce3f1d32dbdbe61ee5b7b9bb2121bde85f7e/job-server/src/main/scala/spark/jobserver/AkkaClusterSupervisorActor.scala#L240

**Comment by noorul** [comment]
@marcoboi Can I close this as we have #917 which also captures the same issue?

**Comment by marcoboi** [comment]
@noorul, not sure closing the ticket is the appropriate action here. 
Issues #916 and #917 are different issues and, in principle, could be dealt with separately. I proposed a change that will address both issues in #918, but that is only the way I addressed the issue.
In fact, if my PR is not merged, someone else might address the two issues separately.
I would recommend keeping this open and separate from #917.
